judge/models/tests/test_contest.py

```diff
@@ -717,8 +707,7 @@ def test_organization_private_contest_methods(self):
             'normal': {
                 'can_see_own_scoreboard': self.assertTrue,
                 'can_see_full_scoreboard': self.assertTrue,
-                'is_live_joinable_by': self.assertTrue,
-                'is_spectatable_by': self.assertTrue,
+                'get_join_type': self.assertLive,
```

Comment:
`organization_private_contest` is ongoing. Add `get_join_type` to the other items.